converters/toZigbee.js

```diff
@@ -4252,6 +4256,18 @@ const converters = {
             await entity.read('hvacThermostat', [0x4000], manufacturerOptions.eurotronic);
         },
     },
+    sinope_thermostat_report_local_temperature: {
+        key: ['report_local_temperature'],
+        convertSet: async (entity, key, value, meta) => {
+            const reportLocalTemperatureValues = {0: 'automatic', 1: '', 2: 'outside temp'};
```

Comment:
`outside temp` -> `outside_temperature`

Reply:
I also noticed there is no expose for this.

Reply:
This was not used and was not necessary. I have removed it. I have check all the added methods to be sure that's the only one.